spinalcordtoolbox/deepseg/models.py

```diff
@@ -0,0 +1,46 @@
+# coding: utf-8
+# List of available models.
+
+
+import os
+
+from spinalcordtoolbox import __sct_dir__
+
+
+MODELS = {
+    'cord-t2star': {'url': 'https://osf.io/8nsa6/',
```

Comment:
The other day we said tha the version is "encoded" in the url --> is it correct? (just checking if we do not need a version or date key)

Reply:
fixed in c03c647bd26a13125c6219e6caa49ef777128f30